tests/codegen/issues/issue-110797-enum-jump-same.rs

```diff
@@ -0,0 +1,26 @@
+//@ compile-flags: -O
+//@ min-llvm-version: 18
+
+#![crate_type = "lib"]
+
+pub enum K {
+    A(Box<[i32]>),
+    B(Box<[u8]>),
+    C(Box<[String]>),
+    D(Box<[u16]>),
+}
+
+// CHECK-LABEL: @get_len
+// CHECK: getelementptr inbounds
+// CHECK-NEXT: load
+// CHECK-NEXT: ret i64
+// CHECK-NOT: switch
```

Comment:
```suggestion
// CHECK-NEXT: {{.*}}:
// CHECK-NEXT: getelementptr inbounds
// CHECK-NEXT: load
// CHECK-NEXT: ret i64
```
The CHECK-NOT here probably wouldn't do what you want (as any switch would be before the other instructions, not after).

Generally I'd say it's good practice to match the full sequence with CHECK-NEXT in the cases where this is possible.